src/pages/News/News.js

```diff
@@ -7,6 +7,7 @@ export const News = () => {
   const [error, setError] = useState(null);
   const [isLoading, setIsLoading] = useState(false);
   const [news, setNews] = useState([]);
+  const [searchValue, setValue] = useState("");
```

Comment:
Good practice is to name state with model `item, setItem` so you would have `searchValue, setSearchValue`

Reply:
`searchValue` would come from props